### PR TITLE
Add a case for python-pyvmomi

### PR DESCRIPTION
**Pull request by xiaodwan**
For bz1733540.

Signed-off-by: Xiaodai Wang <xiaodwan@redhat.com>

### CODE REVIEW

**Comment by xiaodwan** [comment]
depends on https://github.com/avocado-framework/avocado-vt/pull/2528

**Comment by xiaodwan** [comment]
(1/1) type_specific.io-github-autotest-libvirt.query_changed_disk_areas: PASS (112.33 s)

**Comment by xiaodwan** [comment]
@chloerh @chunfuwen Please help to review this PR, thanks.

File: v2v/tests/src/pyvmomi/query_changed_disk_areas.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        disk_change_info = conn.query_changed_disk_areas(
+            disk_label='Hard disk 2')
+        if not disk_change_info.changedArea:
+            test.fail('Not found any changes')
+
+        total = 0
+        for change in disk_change_info.changedArea:
+            total += change.length
+
+        logging.info('total change length is %s', total)
```
Comment (chloerh) [inline v2v/tests/src/pyvmomi/query_changed_disk_areas.py:98]: Seems 'total' is an integer. Whatever...

Comment (xiaodwan) [reply v2v/tests/src/pyvmomi/query_changed_disk_areas.py:98]: Yes, it will be converted to string implicitly.

**Comment by chunfuwen** [comment]
check https://bugzilla.redhat.com/show_bug.cgi?id=1733540, finding those feature is still strongly target to v2v, suggest move cfg and python to tp-libvirt/v2v folder.
@kylazhang ,what's your thoughts?

File: v2v/tests/src/pyvmomi/query_changed_disk_areas.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    "Basic pyvmomi test
+
+    1) create a snapshot
+    2) power on the VM, write some fixed length data to
+       the second disk.
+    3) power off the VM, query the changes and compare
+       the lenght of changed area.
+    """
+
+    def safe_power_off(conn):
```
Comment (chunfuwen) [inline v2v/tests/src/pyvmomi/query_changed_disk_areas.py:19]: miss doc string

File: v2v/tests/src/pyvmomi/query_changed_disk_areas.py [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+        power_state = conn.get_vm_summary()['power_state']
+        if power_state != 'poweredOff':
+            conn.power_off()
+
+    vm_name = params.get("main_vm")
+    if not vm_name:
+        test.error('No VM specified')
+
+    vsphere_host = params.get("vsphere_host")
```
Comment (chunfuwen) [inline v2v/tests/src/pyvmomi/query_changed_disk_areas.py:28]: Better add explanation  or comments what purposes are for parameter

File: v2v/tests/src/pyvmomi/query_changed_disk_areas.py [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+    vsphere_user = params.get("vsphere_user")
+    vsphere_pwd = params.get("vsphere_pwd")
+    vm_bootup_timeout = params.get("vm_bootup_timeout ", 300)
```
Comment (chunfuwen) [inline v2v/tests/src/pyvmomi/query_changed_disk_areas.py:31]: use blank line separate parameter as group to allow more readable

File: v2v/tests/src/pyvmomi/query_changed_disk_areas.py [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+    vm_user = params.get("vm_user", 'root')
+    vm_pwd = params.get("vm_pwd")
+    vm_client = params.get("vm_client", 'ssh')
+    vm_port = params.get("vm_port", 22)
+    vm_prompt = params.get("vm_prompt", r"[\#\$\[\]%]")
+
+    try:
```
Comment (chunfuwen) [inline v2v/tests/src/pyvmomi/query_changed_disk_areas.py:38]: Do you need handle exception for connection? I just observe there is finally block in the end

Comment (xiaodwan) [reply v2v/tests/src/pyvmomi/query_changed_disk_areas.py:38]: No need for this case. Any exception should fail the case and the env should be cleaned up.

File: v2v/tests/src/pyvmomi/query_changed_disk_areas.py (hunk 4 above)
Comment (xiaodwan) [reply v2v/tests/src/pyvmomi/query_changed_disk_areas.py:31]: done

File: v2v/tests/src/pyvmomi/query_changed_disk_areas.py (hunk 3 above)
Comment (xiaodwan) [reply v2v/tests/src/pyvmomi/query_changed_disk_areas.py:28]: done

File: v2v/tests/src/pyvmomi/query_changed_disk_areas.py (hunk 2 above)
Comment (xiaodwan) [reply v2v/tests/src/pyvmomi/query_changed_disk_areas.py:19]: done

**Pull request merged**